yearn/package.json

```diff
@@ -13,24 +29,16 @@
     "ethers": "^5.3.0",
     "hardhat": "^2.3.0",
     "hardhat-contract-sizer": "^2.0.3",
+    "hardhat-deploy": "^0.8.11",
     "hardhat-gas-reporter": "^1.0.4",
     "prettier": "^2.3.0",
     "prettier-plugin-solidity": "^1.0.0-beta.11 ",
     "solhint": "^3.3.6",
     "solhint-config-keep": "github:keep-network/solhint-config-keep",
-    "typescript": "^4.3.2"
+    "ts-node": "^10.1.0",
+    "typescript": "^4.3.5"
   },
-  "license": "MIT",
-  "name": "@keep-network/tbtc-v2-yearn",
-  "scripts": {
-    "build": "hardhat compile",
-    "lint": "npm run lint:js && npm run lint:sol",
-    "lint:fix:js": "eslint . --fix",
-    "lint:fix:sol": "solhint 'contracts/**/*.sol' --fix && prettier --write '**/*.sol'",
-    "lint:js": "eslint . ",
-    "lint:sol": "solhint 'contracts/**/*.sol' && prettier --check '**/*.sol'",
-    "size-contracts": "hardhat compile && hardhat size-contracts",
-    "test": "hardhat test",
-    "test:system": "NODE_ENV=system-test hardhat test ./test/system/*.test.js"
+  "engines": {
+    "node": ">= 12.0.0"
```

Comment:
Can we switch to node 14 as this is the current LTS version?

Reply:
https://github.com/keep-network/tbtc-v2/pull/14/commits/efddb833fde05f4f67c27d8b43e611854c182a7a

Reply:
We also have to update it in README.md, as the node version is mentioned there too.

Reply:
https://github.com/keep-network/tbtc-v2/pull/14/commits/3c1901a27a86d1587e0d3c5c2180944be678da45